libvast/CMakeLists.txt

```diff
@@ -1,162 +1,157 @@
+    src/detail/fdinbuf.cpp
+    src/detail/fdistream.cpp
+    src/detail/fdostream.cpp
+    src/detail/fdoutbuf.cpp
+    src/detail/fill_status_map.cpp
+    src/detail/line_range.cpp
+    src/detail/make_io_stream.cpp
+    src/detail/mmapbuf.cpp
+    src/detail/posix.cpp
+    src/detail/process.cpp
+    src/detail/string.cpp
+    src/detail/system.cpp
+    src/detail/terminal.cpp
+    src/die.cpp
+    src/error.cpp
+    src/event.cpp
+    src/event_types.cpp
+    src/ewah_bitmap.cpp
+    src/expression.cpp
+    src/expression_visitors.cpp
+    src/filesystem.cpp
+    src/format/bgpdump.cpp
+    src/format/csv.cpp
+    src/format/json.cpp
+    src/format/mrt.cpp
+    src/format/multi_layout_reader.cpp
+    src/format/reader.cpp
+    src/format/single_layout_reader.cpp
+    src/format/test.cpp
+    src/format/writer.cpp
+    src/format/zeek.cpp
+    src/http.cpp
+    src/ids.cpp
+    src/json.cpp
+    src/meta_index.cpp
+    src/null_bitmap.cpp
+    src/operator.cpp
+    src/pattern.cpp
+    src/port.cpp
+    src/row_major_matrix_table_slice_builder.cpp
+    src/schema.cpp
+    src/segment.cpp
+    src/segment_builder.cpp
+    src/segment_store.cpp
+    src/store.cpp
+    src/subnet.cpp
+    src/subset.cpp
+    src/synopsis.cpp
+    src/synopsis_factory.cpp
+    src/system/accountant.cpp
+    src/system/application.cpp
+    src/system/archive.cpp
+    src/system/configuration.cpp
+    src/system/connect_to_node.cpp
+    src/system/default_application.cpp
+    src/system/default_configuration.cpp
+    src/system/dummy_consensus.cpp
+    src/system/evaluator.cpp
+    src/system/exporter.cpp
+    src/system/importer.cpp
+    src/system/index.cpp
+    src/system/indexer.cpp
+    src/system/indexer_stage_driver.cpp
+    src/system/node.cpp
+    src/system/partition.cpp
+    src/system/profiler.cpp
+    src/system/query_processor.cpp
+    src/system/query_supervisor.cpp
+    src/system/raft.cpp
+    src/system/remote_command.cpp
+    src/system/signal_monitor.cpp
+    src/system/sink_command.cpp
+    src/system/source_command.cpp
+    src/system/spawn_archive.cpp
+    src/system/spawn_arguments.cpp
+    src/system/spawn_consensus.cpp
+    src/system/spawn_exporter.cpp
+    src/system/spawn_importer.cpp
+    src/system/spawn_index.cpp
+    src/system/spawn_indexer.cpp
+    src/system/spawn_node.cpp
+    src/system/spawn_or_connect_to_node.cpp
+    src/system/spawn_profiler.cpp
+    src/system/spawn_sink.cpp
+    src/system/spawn_source.cpp
+    src/system/start_command.cpp
+    src/system/table_indexer.cpp
+    src/system/task.cpp
+    src/system/tracker.cpp
+    src/system/version_command.cpp
+    src/table_slice.cpp
+    src/table_slice_builder.cpp
+    src/table_slice_builder_factory.cpp
+    src/table_slice_factory.cpp
+    src/time.cpp
+    src/timestamp_synopsis.cpp
+    src/to_events.cpp
+    src/type.cpp
+    src/uuid.cpp
+    src/value.cpp
+    src/value_index.cpp
+    src/value_index_factory.cpp
+    src/view.cpp
+    src/wah_bitmap.cpp)
```

Comment:
This change breaks our [`add_type`](https://github.com/vast-io/vast/blob/master/scripts/add_type) script. I'm using this quite extensively, so to me that's more important than having a coherent CMake style TBH. 😕

Reply:
Maybe we can settle on excluding such `src` blocks via `format off/on` comments?

Reply:
This fails only because the closing ')' is on the same line as the last parameter, right? Why don't we just extend to tools parsing logic to accept that style.